cmd/init.go

```diff
@@ -41,6 +43,7 @@ func createInitCommand(initializer tp.Initializer) *cobra.Command {
     address: "127.0.0.1"
     grpc_port: 19090
     http_port: 19091
+  home_path: "%s"
```

Comment:
@rootwarp 
how about vatz_home_path? 
what do you think?

Reply:
@xellos00 That's better, I'll stack the new commit soon. 👍

Reply:
> @xellos00 That's better, I'll stack the new commit soon. 👍

@rootwarp 
any update?

Reply:
@xellos00 Oh; Sorry, I missed this.
I push the new commit. can you please check this again?

Reply:
> @xellos00 Oh; Sorry, I missed this. I push the new commit. can you please check this again?

Great!!